src/utils/utils.js

```diff
@@ -5,12 +5,12 @@ export const consolePrefix = 'SweetAlert2:'
  * @param arr
  */
 export const uniqueArray = (arr) => {
-  const result = []
-  for (var i in arr) {
-    if (result.indexOf(arr[i]) === -1) {
-      result.push(arr[i])
+  var result = []
+  arr.map(i => {
```

Comment:
`.map` works here, but it's weird because it's not being used as intended. You are not actually "mapping" an array here, so I think `.forEach` is better.

Reply:
Yes but since it's just a one-shot iteration (there's no chaining, etc), a for..of would be better than a foreach (one call on the stack per element).

Reply:
Also, since `i` is more a variable name we use for indexes, I'd prefer `elem` / `element` / `entry` / etc. here.

Reply:
Yes, for..of is even better